index.html

```diff
@@ -99,7 +99,7 @@ <h3 id="features-title">Main Features</h3>
         <div class="container container-double">
             <div class="row is-flex">
                 <div class="col-md-9 feature-detail" id="lightweight">
-                    <h3 style="font-weight: bold">NewPipe - the smart streaming solution</h3>
+                    <h3 style="font-weight: bold">NewPipe—against the stream of the current</h3>
```

Comment:
```suggestion
                    <h3 style="font-weight: bold">NewPipe — against the current of the stream</h3>
```

Reply:
No spacing is how it is used in https://hosted.weblate.org/translate/newpipe/strings/de/?checksum=f34666a9044eb73d
Amounts to choice of style I think.

Reply:
I've only seen the space being absent when you want to show interruption.

"Hey! How are you doin-- whoa!"

(pretend that's an em dash)

Reply:
I think it is either side or nothing.

Reply:
According to that website, the em dash shouldn't even be used here. I think it should be a colon.

Reply:
I mean kinda unrelated but maybe against streams?

Also, I like Opuses dash

Reply:
@opusforlife2 The change doesn't carry over the duality. current works with stream, and can either be in a stream as a resistance, or a virtual one.

Reply:
If you're using this metaphorically, then I would need some more clarification.

In conventional usage, however, we use it this way. 'Current' is the direction the stream flows in. You go against the current, not the stream itself.